src/notation/internal/notationactioncontroller.cpp

```diff
@@ -424,15 +424,16 @@ void NotationActionController::init()
     registerAction("add-down-bow", &Interaction::toggleArticulation, mu::engraving::SymId::stringsDownBow);
     registerAction("transpose-up", &Interaction::transposeSemitone, 1, PlayMode::PlayNote);
     registerAction("transpose-down", &Interaction::transposeSemitone, -1, PlayMode::PlayNote);
-    //registerAction("toggle-insert-mode", );
```

Comment:
This would cause a conflict with #11854, which is currently waiting to be tested

Reply:
Thanks, I reverted this change